latex_files/Makefile

```diff
@@ -0,0 +1,17 @@
+SHELL = /bin/sh
+
+all: main.pdf
+
+main.pdf: *.tex *.bib
+	pdflatex main.tex
+	bibtex main
+	touch main.tex
+	pdflatex main.tex
+	pdflatex main.tex
```

Comment:
In my tests with a full dissertation-length document, page numbers would mis-render by one, but only inconsistently, without a fourth run of pdflatex. Thus, I'm planning to add that in here (as it won't hurt anything to do so, in any case).